e2e/__tests__/__snapshots__/emptyDescribeWithHooks.test.js.snap

```diff
@@ -1,138 +0,0 @@
-// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
lost?

Reply:
The test doesn't run on on jasmine. It has `skipSuiteOnJasmine();`

Reply:
Ah! Do `JEST_CIRCUS=1 yarn jest emptyDescribeWithHooks`

Reply:
Yup! That worked